BangazonAPI/Models/OrderProduct.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class OrderProduct
+    {
+
+        public int Id { get; set; }
+        public int OrderId { get; set; }
```

Comment:
The user is never interacting directly with this table, so we don't need a model for it.

Reply:
Deleted